src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -91,6 +92,11 @@ private void addFolderCoversheet(SortableBundleItem item) throws IOException {
         private void addDocument(SortableBundleItem item) throws IOException {
             PDDocument newDoc = PDDocument.load(documents.get(item));
             merger.appendDocument(document, newDoc);
+
+            if (bundle.getPaginationStyle() != PaginationStyle.off) {
```

Comment:
will it default to null or PaginationStyle.off?

Reply:
I have set the default on Bundle objects to 'off'. Will commit a change shortly